src/locales/en.json

```diff
@@ -154,7 +154,8 @@
     "included": "Included",
     "account-number": "Account number",
     "total-balance": "Total",
-    "deletion_error": "An error occurred while group deletion."
+    "deletion_error": "An error occurred while group deletion.",
```

Comment:
There is a rule between underscore and dash naming ? (deletion_error ; no-account)

Reply:
No rules, but I think we use the dash-between-words more than underscore_between_words. It could be good to change this.

Reply:
https://github.com/cozy/cozy-banks/pull/1975